ui/pages/confirmations/hooks/useLedgerConnection.ts

```diff
@@ -22,15 +23,18 @@ import { useConfirmContext } from '../context/confirm';
 
 const useLedgerConnection = () => {
```

Comment:
Do these changes warrant a quick unit test?

Reply:
As this is [rushed into RC](https://consensys.slack.com/archives/C029JG63136/p1727100953238569?thread_ts=1721989273.481959&cid=C029JG63136) I will merge this and handle tests on different PR